ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -581,15 +587,26 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
             tap(status => this.validStatusChanged.emit(status))
         );
 
+        // We need to trigger an unsubscribe, but don't want to do it prematurely. Wait til we are not busy!
+        combineLatest([this.destroyComponentSignal$, addressComponentBusy$]).pipe(
+            filter(([_, busy]) => !busy),
+            take(1))
+            .subscribe(() => {
+                this.ngUnsubscribe.next();
+                this.ngUnsubscribe.complete();
+            });
+
```

Comment:
This is the unsubscribe that most of the code in this component depends on.
(we were having a problem sometime with either the unsubscribe not occurring at all and sometimes happening prematurely, while saving or cleaning up was still going on). More on this below.